.github/workflows/ci.yaml

```
@@ -0,0 +1,14 @@
+name: CI
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Tests
+    uses: theforeman/actions/.github/workflows/test-gem.yml@v0
+    with:
+      command: bundle exec rake test
```

And also run acceptance tests
```suggestion
      command: bundle exec rake test test:acceptance
```